Person[] persons
uint32 image_w
uint32 image_h
Header header